apps/wallet-mobile/src/yoroi-wallets/cardano/api/metadata.ts

```diff
@@ -43,12 +43,16 @@ export const parseNFT = (
   const nameHex = toAssetNameHex(tokenId)
   const displayAssetName = toDisplayAssetName(tokenId)
```

Comment:
@MichalSzorad `toDisplayAssetName` will remove the `tag` if CIP67, I think the API returns the name as it is (with tags). I think it needs to use `toAssetName`

Reply:
the tag is in hex format, but ascii doesn't have tag in the ids if i am not wrong